android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -186,17 +188,19 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
     }
 
     @ReactMethod
-    override fun registerUser(identityProviderId: String?, scopes: ReadableArray, promise: Promise) {
+    fun registerUser(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
```

Comment:
missing nullcheck

Reply:
This one is supposed to be nullable, as it will register with the default identityprovider

Reply:
focused so much on nullability that I didn't notice that here it actually can be null, sorry :)